kong/runloop/handler.lua

```diff
@@ -777,6 +778,29 @@ return {
       var.upstream_x_forwarded_proto = forwarded_proto
       var.upstream_x_forwarded_host  = forwarded_host
       var.upstream_x_forwarded_port  = forwarded_port
+
+      -- clear hop-by-hop request headers:
+      local connection = var.http_connection
+      if connection then
+        local header_names = regex.split(connection .. ",", [[\s*,\s*]], "djo")
+        if header_names then
+          for i=1, #header_names do
+            if header_names[i] ~= "" and lower(header_names[i]) ~= "close" then
+              ngx.req.clear_header(header_names[i])
+            end
+          end
+        end
+      end
+
+      ngx.req.clear_header("Keep-Alive")
```

Comment:
localize `ngx.req.clear_header`?

Reply:
@p0pr0ck5, the `ngx` is localized, so I just skipped localizing the rest. I can change if that is recommended. I am not sure about it, do you know if there are any benefits of doing so (when the global access is already avoided).

Reply:
Ah heck, i didn't realize the `ngx` is localized. Ignore this :upside_down_face: